modules/PhysiCell_MultiCellDS.cpp

```diff
@@ -890,6 +890,29 @@ void add_PhysiCell_cells_to_open_xml_pugi_v2( pugi::xml_document& xml_dom, std::
 		data_start_indices.push_back( index ); 
 		cell_data_size += size; 
 		index += size; 
+
+        // generation
+		name = "generation"; 
```

Comment:
when merging with dev-paul, will need (or maybe just want?) to update these with Paul's condensed version of this code there